README.md

```diff
@@ -1,7 +1,10 @@
 ## Quickstart
 
 1. Install the dependencies with `npm install`
-2. Run the application with `npm run dev`
+2. Run `rm -rf node_modules/.vite/deps` on a git terminal
+3. Create a .env file on the project base folder
+4. Add the following line to the .env file : `VITE_FIREBASE_TOKEN=AIzaSyDmZ2F2tEqDrZy0ONKRkNYIEk4_piTUAWI`
```

Comment:
Perhaps the FIREBASE TOKEN shouldn't be here as it will be exposed to others, causing a security issue.

Reply:
Yes, 100% we will need to remove this from the commit history as well